mathesar/tests/integration/test_columns.py

```diff
@@ -8,3 +8,26 @@ def test_add_column(page, go_to_patents_data_table):
     page.click("button:has-text('Add')")
     column_header = f".table-content .header .cell .name:has-text('{column_name}')"
     expect(page.locator(column_header)).to_be_visible()
+
+
+def test_convert_text_col_of_num_to_num_col(page, patent_schema, base_schema_url):
+    page.goto(base_schema_url)
+    page.click("[aria-label='New Table']")
+    page.click("button:has-text('Empty Table')")
+    page.click("[aria-label='New Column']")
+    page.fill(".new-column-dropdown input", 'foo')
+    page.click("button:has-text('Add')")
+    page.click(".sheet-cell .cell-wrapper.multi-line-truncate")
+    page.dblclick(".cell-wrapper.multi-line-truncate")
```

Comment:
As a general advice, the playwright codegen tool only generates code based on the recorded user action, which do not usually provide good targeted tests. We should not be using the generated code directly but rather as a base to write our tests upon.

In this case, there is no need for `page.click`, just the `page.dblclick` line would suffice.